.storybook/utils/resetGlobals.js

```diff
@@ -28,7 +28,6 @@ import dashboardData from '../__fixtures__/_googlesitekitLegacyData';
 
 export const resetGlobals = () => {
 	global._googlesitekitLegacyData = cloneDeep( dashboardData );
-	global._googlesitekitLegacyData.admin.assetsRoot = '';
 	global._googlesitekitLegacyData.isStorybook = true;
```

Comment:
Can we remove this one too? I know it's not in the ACs but it's not used anywhere else either and we shouldn't start using it now! 😄 
```suggestion
```